lib/control-app.js

```diff
@@ -28,7 +42,7 @@ module.exports = function(config) {
     config.builder = config.builder || '../lib/builder-browserify';
 
     if (config.builder) {
-        build = require(config.builder);
+        build = require(config.builder)();
```

Comment:
@rase- This is the builder API change, so that multiple instances of control-app can start with different builder "instances". This coupled with the updated browserify-builder, fixes the PR tests